2023/10/21/main.go

```diff
@@ -0,0 +1,17 @@
+// Daily Joel 2023-10-21
+//
+// Slices and range form for loops.
+
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	letters := []string{"J", "o", "e", "l"}
+	for _, l := range letters {
+		fmt.Print(l)
+	}
```

Comment:
for loop going through the range of the slice printing every string at each index after each other